data-es2016plus.js

```diff
@@ -5776,6 +5776,39 @@ exports.tests = [
           reactnative0_70_3: true,
           rhino1_7_13: false
         }
+      },
+      {
+        name: 'resolving identifier in parent scope',
+        exec: function () {/*
+          {
+            let a = ["hello world"];
+            class MyClass {
+              c = a[(0)]; 
+            }
+            return new MyClass().c === a[0];
+          }
+        */},
+        res: {
+          babel6corejs2: true,
+          tr: true,
+          typescript1corejs2: true,
+          ie11: false,
+          firefox2: false,
+          firefox67: firefox.classFields,
+          firefox69: true,
+          opera10_50: false,
+          chrome65: chrome.harmony,
+          chrome72: true,
+          safari15: false,
+          safari16: true,
+          safaritp: true,
+          duktape2_0: false,
+          graalvm19: false,
+          graalvm20: true,
+          hermes0_7_0: false,
+          reactnative0_70_3: true,
+          rhino1_7_13: false
+        }
```

Comment:
Here, I have copy/pasted the data from 'instance class fields / public instance class fields', and then manually modified the entries for safari. Is that the correct approach? Or is there some automated system to collect this data?

Reply:
Ideally, results should only be present in the file if a human has manually verified them.

Reply:
Ok that makes sense. Updated to only include browsers I have manually tested